doc/source/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.rst

```diff
@@ -0,0 +1,39 @@
+.. _PXC-8.0.27-17.1:
+
+================================================================================
+*Percona XtraDB Cluster* 8.0.27-17.1
+================================================================================
+
+:Date: 2022
+:Installation: `Installing Percona XtraDB Cluster <https://www.percona.com/doc/percona-xtradb-cluster/8.0/install/index.html>`_
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+Release Highlights
+================================================================================
+
+The following lists a number of the bug fixes for *MySQL* 8.0.27, provided by Oracle, and included in Percona Server for MySQL:
+
+* The ``default_authentication_plugin`` is deprecated. Support for this plugin may be removed in future versions. Use the ``authentication_policy`` variable.
+* The ``binary`` operator is deprecated. Support for this operator may be removed in future versions. Use ``CAST(... AS BINARY)``.
+* Fix for when a parent table initiates a cascading ``SET NULL`` operation on the child table, the virtual column can be set to NULL instead of the value derived from the parent table.
+
+Find the full list of bug fixes and changes in the `MySQL 8.0.27 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-27.html>`__.
+
+Bugs Fixed
+=======================
+
+* :jirabug:`PXC-3729`: Allowed certified high-priority transactions to proceed without lock conflicts.
+* :jirabug:`PXC-3766`: Stopped every XtraBackup-based SST from executing the version-check procedure.
+* :jirabug:`PXC-3704`: The maximum writeset size is based on ``repl.max_ws_size`` when both ``repl.max_ws_size`` and ``wsrep_max_ws_size`` are passed during startup.
+
+Contact Us
+=======================
+
+The `Documentation Contribution Guide <https://github.com/percona/percona-server/blob/8.0/doc/source/contributing.md>`__ describes the methods available to contribute to the Percona Server for MySQL documentation.
```

Comment:
Change to https://github.com/percona/percona-xtradb-cluster/blob/8.0/doc/source/contributing.md

Reply:
Updated in Useful-links.txt file